baselines/dom.generated.d.ts

```diff
@@ -12642,6 +12646,51 @@ declare var Request: {
     new(input: RequestInfo, init?: RequestInit): Request;
 };
 
+interface ResizeObservation {
+    readonly lastReportedSizes: ReadonlyArray<ResizeObserverSize>;
+    readonly observedBox: ResizeObserverBoxOptions;
+    readonly target: Element;
+}
+
+declare var ResizeObservation: {
+    prototype: ResizeObservation;
+    new(target: Element): ResizeObservation;
+};
+
+interface ResizeObserver {
+    disconnect(): void;
+    observe(target: Element, options?: ResizeObserverOptions): void;
+    unobserve(target: Element): void;
+}
+
+declare var ResizeObserver: {
+    prototype: ResizeObserver;
+    new(callback: ResizeObserverCallback): ResizeObserver;
+};
+
+interface ResizeObserverEntry {
+    readonly borderBoxSize: ReadonlyArray<ResizeObserverSize>;
+    readonly contentBoxSize: ReadonlyArray<ResizeObserverSize>;
+    readonly contentRect: DOMRectReadOnly;
+    readonly devicePixelContentBoxSize: ReadonlyArray<ResizeObserverSize>;
```

Comment:
We also want to remove this specific property `devicePixelContentBoxSize` as it has no browser support. `removedTypes.json` may include 

```json
"ResizeObserverEntry": {
  "properties": {
    "property": {
      "devicePixelContentBoxSize": null
    }
  }
}`.

Reply:
Is it possible to mark it as optional? Or we don't want to have such properties until they come to spec and if someone wants to use them - they can use declaration merging for that?

Reply:
> they can use declaration merging for that?

Yup. It's safer not to include it as unsupported properties are considered as highly experimental (which means it can be in a totally different form when added to browsers)

Reply:
Agree. Done in 1601cca672e5e70a28355deecc13ac6cf5800d90